src/lib/exporter.js

```diff
@@ -16,7 +16,21 @@ module.exports = {
 
     // Remove all the components that are being injected by aframe-editor or aframe
     // @todo Use custom class to prevent this hack
-    Array.prototype.forEach.call(xmlDoc.querySelectorAll('.a-enter-vr,.a-orientation-modal,.Panel,.aframe-editor,.rs-base,.a-canvas,.a-enter-vr-button,style[data-href="style/rStats.css"],style[data-href^="src/panels"],style[data-href="style/aframe-core.css"],link[href^="https://maxcdn.bootstrapcdn.com"]'), function (el) {
+    Array.prototype.forEach.call(xmlDoc.querySelectorAll([
```

Comment:
i guess this way is faster than a `for loop`?

also does the `Array.prototype.forEach` on the `NodeList` work okay on firefox?